rkrishn7/couchsync
# Feat/change username

### 📃 Summary
Created new 'settings' tab where a user's name and avatar can be changed. The user can change their name as of right now.

### 🔍 What's the new behavior?
See above

### 🏷 Task Link

### ↩️ Depends On

Link any other PRs here.

### 🧪 QA

- List the steps to test the changes here.
1. Connect/join a party
2. Open the chat window
3. Type a message
4. Click on the settings tab
5. Change name, a message saying your name has changed will appear
6. Type a new message, notice your name has changed

### 🛑 Risk Analysis

- [ ] Does it update dependencies?
